### PR TITLE
Add a matplotlibrc

### PR DESCRIPTION
**Pull request by drvinceknight**
EDIT: ~~I have also added some comments to ensure we do not reorder these
imports.~~

### CODE REVIEW

**Comment by marcharper** [comment]
How does mpl know to look for the rc file there?

**Comment by drvinceknight** [comment]
> How does mpl know to look for the rc file there?

It automatically picks it up from the dir the tests are run from (https://matplotlib.org/users/customizing.html#the-matplotlibrc-file), so this does not change overall behavior of the library but just ensures the correct setting for CI.

**Comment by marcharper** [comment]
Great, thanks for the explanation.

File: axelrod/tests/unit/test_plot.py [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 from numpy import mean
-import tempfile
 
-import matplotlib
-matplotlib.use("agg")
-
-import matplotlib.pyplot as plt
+import axelrod
 
+import matplotlib
```
Comment (marcharper) [inline axelrod/tests/unit/test_plot.py]: should this one occur before `import axelrod`?

**Comment by marcharper** [comment]
If it works then it's fine with me. Might want to put in a comment somewhere that explains what's happening so someone doesn't move things around later, if that still matters with the rc file.

File: axelrod/tests/unit/test_plot.py (hunk 1 above)
Comment (drvinceknight) [reply axelrod/tests/unit/test_plot.py]: Import orders don't actually matter anymore, the `matplotlibrc` ooverrides everything but I got myself confused there and will reorder to fit PEP8! (and a bonus rerun of CI to make sure it's still working :crossed_fingers:)

**Comment by drvinceknight** [comment]
> Might want to put in a comment somewhere that explains what's happening so someone doesn't move things around later, if that still matters with the rc file.

The import orders no longer matter (the rc file sets it all) but I'll add a comment to the top of the rc file so we now not to delete it :)

**Pull request merged**